src/app/domainregister/domainregister.component.ts

```diff
@@ -210,6 +213,9 @@ export class DomainRegisterComponent implements AfterViewInit {
 
   public check_avail = function () {
     if (this.is_btn_search_domain_disabled) { return; }
+    if ( this.domain_quota_used && this.domain_quota_allowed && this.domain_quota_used >= this.domain_quota_allowed ) {
+        return this.show_error('You have reached your domain quota allowed. Please purchase more Email Hosting products', 'Dismiss');
```

Comment:
We should say "email domain quota" to avoid confusion here.

Suggested text:

`You have reached your allowed Email Domain quota. Please purchase more Email Hosting products.`